pages/api/studentPortal/quizzes/scoringLogicFPL.ts

```diff
@@ -0,0 +1,26 @@
+import { LanguageQuizResultsMap } from "../../../../components/resources/quizzes/langQuiz/LanguageResults";
+import { QuizViewState } from "../../../../components/resources/quizzes/shared/types";
+
+
+export const computeScore = (quizViewState: QuizViewState) => {
+const score = { JavaScript: 0, "HTML/CSS": 0, Python: 0 };
+  
+  quizViewState.questions.map((question) => {
+    question.options.map((option) => {
```

Comment:
use a foreach instead of a mp in this line and the line above. Using map implies that you're creating a new transformed array. It's weird to see a map being used and then you don't do anything with the array. 

What you're looking to do is iterate through the arrays and add to the score variable. Since you're `mutating/editing` the score variable you should use `foreach`